goss-testing/automated/livecd-preflight-checks

```diff
@@ -29,9 +29,7 @@ set -e
 
 # GOSS_BASE should already be set on the PIT node, but just in case we
 # will set it here
-[[ -z ${GOSS_BASE} ]] && export GOSS_BASE="/opt/cray/tests/install/livecd"
-
-export GOSS_LOG_BASE_DIR=/opt/cray/tests/install/logs
+export GOSS_BASE=${GOSS_BASE-"/opt/cray/tests/install/livecd"}
```

Comment:
If GOSS_BASE is null do we really want it to still be null with this assignment? Otherwise I think you want parameter expansion of :- not - which if set and null substitutes word, ref section 2.6.2 of https://pubs.opengroup.org/onlinepubs/009604499/utilities/xcu_chap02.html

(true of all these scripts not going to mark on each)

General gist is if you're wanting to assign if the var is set otherwise use this if it is null or not set you want :-, - is pretty rare usage wise especially for an export.

Reply:
Good catch!

Reply:
I made changes to address this (in all of the files). Please mark this resolved if you think it covers it.